src/version.h.in

```diff
@@ -7,5 +7,11 @@
 #define TIMESCALEDB_MINOR_VERSION "@PROJECT_VERSION_MINOR@"
 #define TIMESCALEDB_PATCH_VERSION "@PROJECT_VERSION_PATCH@"
 #define TIMESCALEDB_MOD_VERSION "@VERSION_MOD@"
+#define OS_NAME "@CMAKE_SYSTEM_NAME@"
```

Comment:
AFAIK, this is only setting the `OS_NAME` and `OS_VERSION` the source was compiled on. There's no guarantee that these line up with systems the binary runs on, in particular the `OS_VERSION`. Not saying that this is useless info, but not sure it gives the thing you want. Also, perhaps this should be `BUILD_OS_NAME` and `BUILD_OS_VERSION` for clarity?

Reply:
done